jupyterlab_comments/README.md

```diff
@@ -55,5 +55,11 @@ jupyter labextension uninstall jupyterlab-comments
 Add the following lines to your Jupyter config file (e.g. jupyter_notebook_config.py):
 ```bash
 c.Git.remote = '${NAME_OF_REMOTE_REPOSITORY}'
```

Comment:
nit: Example: do NAME of Repo is the Repo URL? (Example: https://github.com/GoogleCloudPlatform/ai-platform-samples.git) ?

Reply:
This is intended to be the short name of the remote repository to push to (if the user has more than one remote repository configured), such as 'origin' or 'upstream' -- like the output of the 'git remote' command. I'll change the README so it's more clear!